### PR TITLE
TFW-4082 - moved bookmark button to the search inputs row

### PR DESCRIPTION
**Pull request by troyblank**
This is just to MOVE the bookmark button, there is another story to handle the new styles of the button I will take on next.

Screen Shot 2019-08-08 at 12 06 41 PM

### CODE REVIEW

File: components/universalSearch/searchBar.js [hunk 1]
```diff
@@ -131,6 +132,38 @@ class SearchBar extends PureComponent {
     //     return match.attributes || ['autocomplete'];
     // }
```
Comment (jasonhale) [inline components/universalSearch/searchBar.js]: Would you mind terribly removing those comments above?  We don't need those anymore.  Or we can do in a different commit.  No bigs.

Comment (troyblank) [reply components/universalSearch/searchBar.js]: Yea there is tons of these that need to be removed; considering it's not really part of this work I feel we need just a general cleanup commit at this point there is probably half of them no longer needed for reference. I will hold off for now but will get this in my next PR.
